pkg/cloud/gen/main.go

```diff
@@ -309,8 +309,13 @@ func (m *Mock{{.Service}}Obj) ToGA() *{{.GA.FQObjectType}} {
 	}
 }
 
+func shouldAddIdToCreateCall(obj string) bool {
```

Comment:
Can we document this to make it clear to future maintainers?

I also suggest changing the name to make it easier to understand.

```
// callOperationRequiresID returns true if the ServiceInfo.Object is
// of a particular type.
func callOperationRequiresID(objectName string) bool 
```

Reply:
done, I added couple more types I found